### DISCUSSION TITLE
Starting contributing

### DISCUSSION THREAD

**Discussion by euotiniel** [angolaosc/.github#6] [Ideas]
I'm very excited about the community, and I almost always stop by to take a look at updates.

But I have two issues:

+ When will the projects start?
+ And what will be the main ones?

Waiting for feedback. 

> And you! Do you have an open source project or are you thinking of starting one?

**Comment by antonio-pedro99** [comment]
Hey @euotiniel You are free to write in Portuguese if you wish to.

Estamos no processo de preparar a nossa agenda. A nossa agenda constará informações sobre as atividades que planeamos organizar. Spoiler, temos planos de organizar uma live de lançamento, onde falaremos mais sobre a comunidade, etc. Enquanto isso, peço-lhe que fique ligado à nossa comunidade para informações futuras. Adicionando, se já tem projetos open-source, ou tem qualquer dúvida sobre open-source, podes usar o nosso servidor no discord.

**Comment by Rubinaldod** [comment]
boa!!!!! ja estava na hora, muito obrigado pela iniciativa!

**Comment by Rubinaldod** [comment]
boa!!!!! ja estava na hora, muito obrigado pela iniciativa!
